src/main/kotlin/racingcar/RandomForceGenerator.kt

```diff
@@ -0,0 +1,11 @@
+package racingcar
+
+import kotlin.random.Random
+
+class RandomForceGenerator : ForceGenerator {
+    private val random: Random = Random
+
+    override fun generate(): Force {
+        return Force(random.nextInt(from = 0, until = 10))
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
        return Force((1..9).random())
```